flycheck.el

```diff
@@ -4849,6 +4849,10 @@ The echo area may be used if the cursor is not in the echo area,
 and if the echo area is not occupied by minibuffer input."
   (not (or cursor-in-echo-area (active-minibuffer-window))))
 
+(define-derived-mode flycheck-error-message-mode fundamental-mode
+  "Flycheck error messages"
+  "Major mode for extended error messages")
```

Comment:
Missing `.` at the end of the docstring.
Also, should the mode derive from text-mode?